shared/cloud-gateway/develop/_send-receive-media-streams-c.mdx

```diff
@@ -0,0 +1,540 @@
+import * as data from '@site/data/variables';
+
+<PlatformWrapper platform="linux">
+This page introduces how to use the Cloud Gateway to send media streams to the client and receive media streams from the client.
+
+## Prerequisites
+
+Before you begin, make sure you have downloaded the latest Cloud Gateway. See [Run the Sample Project](../get-started/integrate-c-sdk).
+
+## Preparation
+
+Complete the following steps before sending and receiving media streams.
+
+### Step 1: Initialize the SDK
+
+You need to call `createAgoraService` and `initialize` to create and initialize an `AgoraService` object. The `IAgoraService` object persists as long as the server app keeps running.
+
+The SDK supports user IDs in both int format and string format. This page only shows user IDs in int format (the character set can only be digits). To learn more about user IDs in string format, see [Using User IDs in String Format](../develop/stringuid-c).
+
+```C++
+// Creates an IAgoraService object
+auto service = createAgoraService();
+// Initializes the IAgoraService object
+agora::base::AgoraServiceConfiguration scfg;
+// Sets Agora App ID
+scfg.appId = appid;
+// Enables the audio processing module
+scfg.enableAudioProcessor = enableAudioProcessor;
+// Disables the audio device module (Normally we do not directly connect audio capture or playback devices to a server)
+scfg.enableAudioDevice = enableAudioDevice;
+// Whether to enable video
+scfg.enableVideo = enableVideo;
+// Disables user IDs in string format (the character can be digits, letters, or special symbols) so that user ID can only contain digits
+scfg.useStringUid = enableuseStringUid;
+if (service->initialize(scfg) != agora::ERR_OK) {
+    return nullptr;
+  }
+```
+
+### Step 2: Connect to the Agora RTC Channel
+
+After initializing the SDK, you can refer to the following steps to connect to the Agora RTC channel.
+
+1. Call `createRtcConnection` to create an `IRtcConnection` object to connect to the Agora RTC channel:
+
+    ```C++
+    // Creates an IRtcConnection object
+    agora::rtc::RtcConnectionConfiguration ccfg;
+    ccfg.autoSubscribeAudio = false;
+    ccfg.autoSubscribeVideo = false;
+    ccfg.clientRoleType = agora::rtc::CLIENT_ROLE_BROADCASTER;
+    agora::agora_refptr<agora::rtc::IRtcConnection> connection = service->createRtcConnection(ccfg);
+    ```
+
+2. Call `registerObserver` to listen to connection events. The `SampleConnectionObserver` class in the sample code inherits the `IRtcConnectionObserver` class and the `INetworkObserver` class:
+
+    ```C++
```

Comment:
Code highlighting looks off. Maybe something is wrong with the rendering.

Reply:
Fixed.